### PR TITLE
Add alternative docker container based on alpine

### PR DESCRIPTION
**Pull request by DuratarskeyK**
Add an alternative Dockerfile based on alpine, that builds faster than ubuntu one, weights less and adds support for proxy.php left/right support as it is on the site itself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ docker build -t jdd:v1 .
 docker run -i --name jdd -p 127.0.0.1:8080:80/tcp jdd:v1
 ```
 
+There is also an alternative Docker container, available as Docker.alpine, based on alpine that includes left/right support, if you want that. To build:
+
+```
+docker build --tag=jdd:v1-alpine --file=Dockerfile.alpine .
```
Comment (zgrossbart) [inline README.md:27]:
```suggestion
docker build --tag=jdd:v1-alpine --file=Dockerfile .
```

**Review by zgrossbart — CHANGES_REQUESTED** [review]
This looks really great.  Thank you so much.  I made one suggested change to the READM to fix the `docker build` command.

**Comment by DuratarskeyK** [comment]
Uh, sorry, I left your original Dockerfile and added my own called Dockerfile.alpine.

**Comment by DuratarskeyK** [comment]
I added a second build-run command for my own Dockerfile.

**Review by zgrossbart — APPROVED** [review]
Looks great.  Thank you.

**Comment by DuratarskeyK** [comment]
docker build --tag=jdd:v1-alpine --file=Dockerfile .
Should be
docker build --tag=jdd:v1-alpine --file=Dockerfile.alpine .
If you want me to make an ubuntu/alpine dockerfile I can try, though I don't have experience in that(but I'll figure it out, I guess).

**Comment by zgrossbart** [comment]
The dockerfile you made is great.  Thank you very much for the contribution.  I don't think we need another one.

**Pull request merged**